app/models/event.rb

```diff
@@ -3,6 +3,8 @@ class Event < ApplicationRecord
   friendly_id :name, use: [:slugged, :finders]
   has_many :rsvps
 
+  validates :description, length: { maximum: 2000 }
```

Comment:
Aye, you’re completely correct of course 👌 

Seeing as this is such a simple app for now and it’s already deployed, I think we’re ok to leave it as-is.